htlcswitch/link.go

```diff
@@ -652,55 +652,48 @@ func (l *channelLink) handleDownStreamPkt(pkt *htlcPacket, isReProcess bool) {
 			// machine, as a result, we'll signal the switch to
 			// cancel the pending payment.
 			default:
+				log.Infof("Unable to handle downstream add HTLC: %v", err)
+
 				var (
-					isObfuscated bool
+					localFailure = false
```

Comment:
Why the change in naming? As a result, the semantics have now been flipped everywhere.

Reply:
It represents something different from what `isObfuscated` used to.